lib/archethic/beacon_chain/update.ex

```diff
@@ -70,7 +70,7 @@ defmodule Archethic.BeaconChain.Update do
   end
 
   def handle_cast({:unsubscribe, node_public_key}, state) do
-    new_state = Map.delete(state, node_public_key)
+    new_state = if node_public_key, do: Map.delete(state, node_public_key), else: Map.new()
```

Comment:
Why do you want to clean if the public key is not the same ?
If some malicious actors send a P2P message to unsubscribe it could remove subscription for all others clients (nodes)